src/main/webapp/app/exercises/shared/participation-submission/participation-submission.component.html

```diff
@@ -17,53 +17,80 @@
             </ng-template>
             <ng-template ngx-datatable-cell-template let-value="value">
                 {{ value | artemisDate: 'long':true }}
             </ng-template>
         </ngx-datatable-column>
-        <ngx-datatable-column prop="results">
+        <ngx-datatable-column prop="results" [minWidth]="60" [width]="80">
             <ng-template ngx-datatable-header-template>
                 <span jhiTranslate="artemisApp.participation.participationSubmission.resultCount"></span>
             </ng-template>
             <ng-template ngx-datatable-cell-template let-value="value">
                 {{ value.length || 0 }}
             </ng-template>
         </ngx-datatable-column>
-        <ngx-datatable-column prop="latestResult">
+        <ngx-datatable-column prop="results" [minWidth]="500" [width]="500">
             <ng-template ngx-datatable-header-template>
-                <span jhiTranslate="artemisApp.participation.participationSubmission.lastResult"></span>
+                <span jhiTranslate="artemisApp.participation.participationSubmission.results"></span>
             </ng-template>
             <ng-template ngx-datatable-cell-template let-value="value">
-                <jhi-result [result]="value" [participation]="participation" [showUngradedResults]="true" [showGradedBadge]="true"></jhi-result>
+                <div class="d-flex flex-column">
+                    <div *ngFor="let result of value">
+                        <jhi-result [result]="result" [participation]="participation" [showUngradedResults]="true" [showGradedBadge]="true"></jhi-result>
+                    </div>
+                </div>
             </ng-template>
         </ngx-datatable-column>
-        <ngx-datatable-column prop="latestResult">
+        <ngx-datatable-column prop="results" [minWidth]="180" [width]="180">
+            <ng-template ngx-datatable-header-template>
+                <span jhiTranslate="artemisApp.result.assessmentType"></span>
+            </ng-template>
+            <ng-template ngx-datatable-cell-template let-value="value">
+                <div class="d-flex flex-column">
+                    <div *ngFor="let result of value">{{ result.assessmentType }}</div>
+                </div>
+            </ng-template>
+        </ngx-datatable-column>
+        <ngx-datatable-column prop="results" [minWidth]="200" [width]="200">
             <ng-template ngx-datatable-header-template>
                 <span jhiTranslate="artemisApp.participation.participationSubmission.assessor"></span>
             </ng-template>
             <ng-template ngx-datatable-cell-template let-value="value">
-                {{ value && value.assessor ? value.assessor.name : 'n.a' }}
+                <div class="d-flex flex-column">
+                    <div *ngFor="let result of value">{{ result.assessor ? result.assessor.name : 'n.a' }}</div>
```

Comment:
In case the name would be undefined, you could use this:

```suggestion
                    <div *ngFor="let result of value">{{ result?.assessor?.name || 'n.a' }}</div>
```

Reply:
Wow thanks I totally forgot about that